apps/functions/register-representation/register-representation-command/index.js

```diff
@@ -0,0 +1,59 @@
+import api from './back-office-api-client.js';
+import { isEmpty, pick } from 'lodash-es';
+
+const mapContactDetails = (entity) => {
+	if (isEmpty(entity)) return {};
+
+	return {
+		...pick(entity, [
+			'firstName',
+			'lastName',
+			'organisationName',
+			'jobTitle',
+			'email',
+			'phoneNumber',
+			'type',
+			'under18'
+		]),
+		...(!isEmpty(entity.address)
+			? {
+					address: pick(entity.address, [
+						'addressLine1',
+						'addressLine2',
+						'town',
+						'county',
+						'postcode',
+						'country'
+					])
+			  }
+			: undefined)
+	};
+};
+
+/**
+ *
+ * @param {import('@azure/functions').Context} context
+ * @param {import('@pins/api/src/message-schemas/commands/register-representation').RegisterRepresentation} msg
+ */
+export default async function (context, msg) {
+	context.log(`Handle register-representation event: ${JSON.stringify(msg)}`);
+
+	const caseId = await api.getCaseID(msg.caseReference);
+	if (!caseId) {
+		throw new Error(`No case found with reference: ${msg.caseReference}`);
```

Comment:
```suggestion
	const caseReference = await api.getCaseID(msg.caseReference);
	if (!caseReference) {
		throw new Error(`No case found with caseReference: ${msg.caseReference}`);
```

Reply:
Just a minor suggestion for clarify but feel free to ignore this.

Reply:
Thanks. I've updated the error message but the variable name is accurate as-is, the function returns `caseId` (Int) rather than `caseReference`(String) that Front Office uses